quill-ndbc-postgres/src/main/scala/io/getquill/context/ndbc/PostgresDecoders.scala

```diff
@@ -1,17 +1,16 @@
 package io.getquill.context.ndbc
 
 import java.time._
-import java.util.{ Date, UUID }
-
-import scala.language.implicitConversions
-import scala.math.BigDecimal.javaBigDecimal2bigDecimal
+import java.util.{Date, UUID}
 
 import io.getquill.context.sql.encoding.ArrayEncoding
-import io.getquill.dsl.LowPriorityPostgresImplicits
 import io.trane.ndbc.PostgresRow
 import io.trane.ndbc.value.Value
 
-trait PostgresDecoders extends LowPriorityPostgresImplicits with io.getquill.dsl.LowPriorityImplicits {
+import scala.language.implicitConversions
+import scala.math.BigDecimal.javaBigDecimal2bigDecimal
+
+trait PostgresDecoders {
```

Comment:
Looks like LowPriorityPostgresImplicits and LowPriorityImplicits extensions only need to be there for the encoders. I'm not sure why.